res/values/strings_tl.arb

```diff
@@ -0,0 +1,681 @@
+  "change_currency" : "Baguhin ang pera",
+  "overwrite_amount" : "I-overwrite ang halaga",
+  "qr_payment_amount" : "Ang QR code na ito ay naglalaman ng halaga ng pagbabayad. Gusto mo bang i-overwrite ang kasalukuyang halaga?",
+
+  "copy_id" : "Kopyahin ang ID",
+  "exchange_result_write_down_trade_id" : "Mangyaring kopyahin o isulat ang trade ID upang magpatuloy.",
+  "trade_id" : "Trade ID:",
+  "copied_to_clipboard" : "Kinopya sa Clipboard",
+  "saved_the_trade_id" : "Nai-save ko na ang trade ID",
+  "fetching" : "Kumukha",
+  "id" : "ID: ",
+  "amount" : "Halaga: ",
+  "payment_id" : "ID ng nagbabayad: ",
+  "status" : "Katayuan: ",
+  "offer_expires_in" : "Mag-e-expire ang mga alok sa: ",
+  "trade_is_powered_by" : "Ang kalakal na ito ay pinatakbo ng ${provider}",
+  "copy_address" : "Kopyahin ang Address",
+  "exchange_result_confirm" : "Sa pamamagitan ng pagpindot sa kumpirmahin, ikaw ay magpapadala ${fetchingLabel} ${from} mula sa inyong wallet na tinatawag ${walletName} sa wallet na ipinapakita sa ibaba. O pwede kang magpadala sa inyong external wallet sa ibabang address/QR code.\n\nPara magpatuloy, mangyaring pindutin upang kumpirmahin o bumalik para baguhin ang halaga.",
+  "exchange_result_description" : "Kailangan mong magpadala ng minimum ${fetchingLabel} ${from} sa address na ipinakita sa susunod na pahinga. Kung magpapadala ka ng halagang mas masmababa sa ${fetchingLabel} ${from} maaring hindi ito ma-convert at maaaring hindi ito ma-refund.",
+  "exchange_result_write_down_ID" : "*Mangyaring kopyahin o isulat ang inyong ID na ipinapakita sa itaas.",
+  "confirm" : "Kumpirmahin",
+  "confirm_sending" : "Kumpirmahin ang pagpapadala",
+  "commit_transaction_amount_fee" : "Gumawa ng transaksyon sa halagang: ${amount}\nFee: ${fee}",
+  "sending" : "Nagpapadala",
+  "transaction_sent" : "Ipinadala ang transaksyon!",
+  "expired" : "Nag-expire na",
+  "time" : "Oras ${minutes}m ${seconds}s",
+  "send_xmr" : "Ipadala anf XMR",
+  "exchange_new_template" : "Bagong template",
+
+  "faq" : "FAQ",
+
+
+  "enter_your_pin" : "Ilagay ang iyong PIN",
+  "loading_your_wallet" : "Naglo-load ng iyong wallet",
+
+
+  "new_wallet" : "Bagong Wallet",
+  "wallet_name" : "Pangalan ng Wallet",
+  "continue_text" : "Magpatuloy",
+  "choose_wallet_currency" : "Mangyaring pumili ng pera sa wallet:",
+
+
+  "node_new" : "Bagong Node",
+  "node_address" : "Node Address",
+  "node_port" : "Node port",
+  "login" : "Mag login",
+  "password" : "Password",
+  "nodes" : "Mga node",
+  "node_reset_settings_title" : "I-reset ang mga settings",
+  "nodes_list_reset_to_default_message" : "Sigurado ka bang gusto mo bang i-reset ang mga settings sa default?",
+  "change_current_node" : "Sigurado ka bang babaguhin ang kasalukuyang node sa${node}?",
+  "change" : "pagbabago",
+  "remove_node" : "Alisin ang node",
+  "remove_node_message" : "Sigurado ka bang gusto mong tanggalin ang napiling node?",
+  "remove" : "Tanggalin",
+  "delete" : "Tanggalin ",
```

Comment:
Just curious, @nahuhh , you marked this as resolved without changing anything: You don't see using the same translation as an issue?

I am afraid we have a case where we e.g. translate *hide* and *delete* with the same word, where the first is pretty harmless because you can unhide at any time, but the second can be quite bad.

Reply:
![of5rk2pqyr27x7bf.jpg](https://user-images.githubusercontent.com/50635951/215191892-928b8cda-9cc1-4270-a5a1-38342cc66370.jpg)

![w09vcro346viw8lu.jpg](https://user-images.githubusercontent.com/50635951/215191915-1591872d-af63-4b9d-a318-0519dd065389.jpg)